client/app/pages/queue/enqueuing.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div>
+    <TitleBlock title="Queue" summary="Where the magic happens.">
+      <template #right>
+        <QueueTitleRight />
+      </template>
+    </TitleBlock>
+
+    <QueueTabs :current-tab="currentTab" @pending="pending" @refresh="refresh" />
+
+    <ErrorAlert v-if="error">
+      {{ error }}
+    </ErrorAlert>
+
+    <div class="p-2">
+      <RpcTable>
+        <RpcThead>
+          <tr v-for="headerGroup in table.getHeaderGroups()" :key="headerGroup.id">
+            <RpcTh
+              v-for="header in headerGroup.headers"
+              :key="header.id"
+              :colSpan="header.colSpan"
+              :is-sortable="header.column.getCanSort()"
+              :sort-direction="header.column.getIsSorted()"
+              @click="header.column.getToggleSortingHandler()?.($event)"
+            >
+              <FlexRender v-if="!header.isPlaceholder" :render="header.column.columnDef.header"
+                :props="header.getContext()" />
+            </RpcTh>
+          </tr>
+        </RpcThead>
+        <RpcTbody>
+          <RpcRowMessage :status="status" :column-count="table.getAllColumns().length" :row-count="table.getRowModel().rows.length" />
+          <tr v-for="row in table.getRowModel().rows" :key="row.id">
+            <RpcTd v-for="cell in row.getVisibleCells()" :key="cell.id">
+              <FlexRender :render="cell.column.columnDef.cell" :props="cell.getContext()" />
+            </RpcTd>
+          </tr>
+        </RpcTbody>
+        <RpcTfoot>
+          <tr v-for="footerGroup in table.getFooterGroups()" :key="footerGroup.id">
+            <RpcTh v-for="header in footerGroup.headers" :key="header.id" :colSpan="header.colSpan">
+              <FlexRender v-if="!header.isPlaceholder" :render="header.column.columnDef.footer"
+                :props="header.getContext()" />
+            </RpcTh>
+          </tr>
+        </RpcTfoot>
+      </RpcTable>
+    </div>
+  </div>
+
+</template>
+
+<script setup lang="ts">
+import { Anchor, Icon } from '#components'
+import {
+  FlexRender,
+  getCoreRowModel,
+  useVueTable,
+  createColumnHelper,
+  getFilteredRowModel,
+  getSortedRowModel,
+  type SortingState,
+} from '@tanstack/vue-table'
+import type { QueueItem } from '~/purple_client'
+import type { TabId } from '~/utils/queue'
+
+const api = useApi()
+
+const currentTab: TabId = 'enqueuing'
+
+const {
+  data,
+  pending,
+  status,
+  refresh,
+  error,
+} = await useAsyncData(
+  `queue2-enqueuing`,
```

Comment:
I don't know where we stand on this as lint, but this doesn't need to be a template literal since it's not interpolated.